### PR TITLE
Benchmarking workflow

### PR DESCRIPTION
**Pull request by mccabete**
This contains small tweaks of the benchmarking workflow.  

## Description
1. fixed a few personal hacks (tess.xml) that made it into my last pull request
2. added entries to the bety_mstmip_lookup.csv. 
3. took out redundant variables in align_data
4. changed load_data & read.output @bcow this is the only bit that I imagine could break your stuff if you want to take a look

## Motivation and Context
Edits to the bety_mstmip_lookup were to ensure that if users used variables that were identical to the mstmip standard names, that the lookup table could accommodate that. 

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/benchmark/R/align_data.R [hunk 1]
```diff
@@ -4,16 +4,14 @@
...
-##' @param end_year numeric
 ##' @return dat
 
 ##' @author Betsy Cowdery
 
 
 ## Align timeseries data using different functions
 
-align_data <- function(model.calc, obvs.calc, var, start_year, end_year, align_method = "match_timestep") {
+align_data <- function(model.calc, obvs.calc, var, align_method = "match_timestep") {
```
Comment (istfer) [inline modules/benchmark/R/align_data.R]: this is going to break [PDA](https://github.com/PecanProject/pecan/blob/develop/modules/assim.batch/R/pda.get.model.output.R#L114), please grep where the function's being called through the code and change their arguments too when you make a change to a fcn arguments.

Comment (mccabete) [inline modules/benchmark/R/align_data.R]: Yep. Can do.

File: utils/R/read.output.R [hunk 2]
```diff
@@ -178,6 +178,7 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
       for (i in seq_along(n)) {
         y <- c(y, rep(years[i], n[i]))
       }
+      model$year<-rep(NA, seq_along(n))
```
Comment (infotroph) [inline utils/R/read.output.R]: Will this change have any effect? I would expect line 182 to promptly clobber this one.

File: modules/benchmark/R/align_data.R (hunk 1 above)
Comment (infotroph) [inline modules/benchmark/R/align_data.R]: I did this grep before I saw istfer's comment, so here are the places I see that could potentially need changing (obviously irrelevant grep hits removed):

```
pecan/documentation/tutorials/AnalyzeOutput/modelVSdata.Rmd:123:aligned_dat = PEcAn.benchmark::align_data(model.calc = model, obvs.calc= observations, var ="NEE", start_year =start_year, end_year = end_year, align_method ="mean_over_larger_timestep")
pecan/modules/assim.batch/R/pda.get.model.output.R:114:    dat <- PEcAn.benchmark::align_data(model.calc = model, obvs.calc = inputs[[k]]$data, var = data.var, 
pecan/modules/benchmark/man/align_data.Rd:7:align_data(model.calc, obvs.calc, var, start_year, end_year,
pecan/modules/benchmark/R/calc_metrics.R:18:  dat <- align_data(model.calc, obvs.calc, var, start_year, end_year, 
```

**Comment by infotroph** [comment]
Uh, maybe "request changes" was the wrong button here -- I think this is OK to merge once the  align_data calls are updated.

File: utils/R/read.output.R (hunk 2 above)
Comment (mccabete) [inline utils/R/read.output.R]: Possible not. I can take this out. I actually cant remember how it got in  in the first place.

**Comment by infotroph** [comment]
I see now that you'd fixed all the align_data calls before I even started reviewing... Definitely the wrong button. Sorry for the noise, this all looks good now.

**Pull request merged**